.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
What is the use of this line?

Reply:
GitHub displays the names of your workflows on your repository's actions page.

Reply:
The template includes a matrix strategy that builds and tests your code with two Node.js versions: 10.x, and 12.x

Reply:
So do you want two version? or if you can change it to only one version which you need

Reply:
. We can build and test with exact versions. Better to have more than one version.

Reply:
How can you do that? Any example script?

Reply:
https://help.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions

Reply:
// We can build and test with exact versions. // You mentioned this. So how can you do? 
comment a script to How to build with that version?

Reply:
I don't know  how to build and test using this

Reply:
So if you don't know, then why you add two versions here? Just go with one version.
` You have to be confident for each line which you writes `

Reply:
Okay. Is it okay to add only one version? Is there is any issue when we add two versions here?

Reply:
why you need two versions for this PoC? I feel only one version is enough. If you hv any specific requirements, then you hv to.
Now no need

Reply:
Okay. Got it

Reply:
Do the changes and push again

Reply:
version means the version of npm of our local machine. Right?

Reply:
node-version: [10.x, 12.x] --> what does it means?